src/main.c

```diff
@@ -6046,6 +6089,9 @@ static long ctrl_ioctl(struct file *filp, unsigned int cmd, unsigned long arg){
 			break;
 		}
 
+		if (snap_devices[minor])
```

Comment:
This check is added because one can try to call transition to incremental on a non-existing snapshot device.